### PR TITLE
[NO-ISSUE] List single files in .rat-excludes aligning with other repos

### PR DESCRIPTION
**Pull request by tkobayas**
List single files in .rat-excludes aligning with other repos

### CODE REVIEW

File: .rat-excludes [hunk 1]
```diff
@@ -1,45 +1,346 @@
...
-.*DMN.*\.xsd
-dmn.xsd
-DC\.xsd
-DI\.xsd
-KiePMML.*\.txt
-TargetValueFactoryTest_01\.txt
-TargetFieldFactoryTest_01\.txt
-.*\.uml
+# drools-decisiontables/src/main/java/org/drools/decisiontable/parser/csv/CsvLineParser.java
+CsvLineParser.java
```
Comment (tkobayas) [inline .rat-excludes:8]: CsvLineParser.java is listed in `LICENSE`

File: .rat-excludes [hunk 2]
```diff
@@ -1,45 +1,346 @@
...
+# drools-decisiontables/src/test/resources/org/drools/decisiontable/unicode.drl.csv
+unicode.drl.csv
+# drools-decisiontables/src/test/resources/org/drools/decisiontable/integrationtests/csv/kmoduleCSVTest.dtable.drl.csv
+kmoduleCSVTest.dtable.drl.csv
+# drools-docs/supplemental-ui/css/search.css
+search.css
+# drools-docs/supplemental-ui/js/search-ui.js
+search-ui.js
+# drools-docs/supplemental-ui/js/vendor/lunr-2.3.9.min.js
+lunr-2.3.9.min.js
```
Comment (tkobayas) [inline .rat-excludes:34]: `drools-docs` is excluded from the source distribution, so no need to check.

File: .rat-excludes [hunk 3]
```diff
@@ -1,45 +1,346 @@
...
+# drools-drl/drools-drl-parser/src/main/antlr4/org/drools/drl/parser/antlr4/JavaLexer.g4
+JavaLexer.g4
+# drools-drl/drools-drl-parser/src/main/antlr4/org/drools/drl/parser/antlr4/JavaParser.g4
+JavaParser.g4
```
Comment (tkobayas) [inline .rat-excludes:38]: JavaLexer.g4, JavaParser.g4 are listed in `LICENSE`

File: .rat-excludes [hunk 4]
```diff
@@ -1,45 +1,346 @@
...
+# kie-dmn/kie-dmn-backend/src/test/resources/DMN14.xsd
+DMN14.xsd
+# kie-dmn/kie-dmn-backend/src/test/resources/DMN15.xsd
+DMN15.xsd
+# kie-dmn/kie-dmn-backend/src/test/resources/DMNDI12.xsd
+DMNDI12.xsd
+# kie-dmn/kie-dmn-backend/src/test/resources/DMNDI13.xsd
+DMNDI13.xsd
+# kie-dmn/kie-dmn-backend/src/test/resources/DMNDI15.xsd
+DMNDI15.xsd
```
Comment (tkobayas) [inline .rat-excludes:126]: xsd files from OMG are listed in `LICENSE`

File: .rat-excludes [hunk 5]
```diff
@@ -1,45 +1,346 @@
...
+# drools-drlonyaml-parent/drools-drlonyaml-todrl/src/main/resources/drl.ftl
+drl.ftl
+# drools-examples/src/main/resources/org/drools/examples/sudoku/Sample.sdo
+Sample.sdo
+# drools-examples/src/main/resources/org/drools/examples/troubleticketWithDSL/TroubleTicketWithDSL.dslr
+TroubleTicketWithDSL.dslr
+# drools-impact-analysis/drools-impact-analysis-graph/drools-impact-analysis-graph-graphviz/src/test/resources/org/drools/impact/analysis/graph/graphviz/simple.dot
+simple.dot
+# drools-model/drools-mvel-parser/src/main/javacc/mvel.jj
+mvel.jj
```
Comment (tkobayas) [inline .rat-excludes:60]: `mvel.jj` is listed in `LICENSE`

**Review by jomarko — APPROVED** [review]
Thank you @tkobayas

**Review by yesamer — APPROVED** [review]

**Review by mariofusco — APPROVED** [review]

**Pull request merged**